packages/cli/src/commands/start.ts

```diff
@@ -82,26 +82,24 @@ async function askQuestions({ template: defaultTemplate }: Pick<StartOptions, "t
 
   // Ask for project directory
   {
-    const projectRelativePath = await p.text({
+    const projectPath = await p.text({
       message: "Enter the project directory path:",
       placeholder: "./my-agentica-project",
       validate(value) {
-        if (value === "") {
+        if (value.trim() === "") {
           return "Please enter a directory path";
         }
-        if (value[0] !== ".") {
```

Comment:
if you want automatic relative path, we can block path startswith `~` and `/` .
this keyword mean absolute path in linux

There is also a window similar to that.